### ISSUE TITLE
Show difficulty star rating when hovering a difficutly in timeshit multiplayer room even when the room is not selected

### ISSUE THREAD

**Issue by Gonzalo-Bruna** [ppy/osu#9665]
**Describe the new feature:** 

in many ocassions, i have wanted to see the difficulty star rating of a room, but if the room is not selected, even if you hover the diff, you can not see its star rating.

Hovering when the room is selected: 
![1](https://user-images.githubusercontent.com/41755816/88416464-455dcc00-cdae-11ea-910d-4b076a829066.jpg)

Hovering when the room is not selected:
![2](https://user-images.githubusercontent.com/41755816/88416480-4c84da00-cdae-11ea-8ae2-470e38834ff9.jpg)

**Proposal designs of the feature:** 

It should show you the diff star rating just like it does right now when the room is selected.

**Comment by Joehuu** [comment]
I've changed the behavior in https://github.com/ppy/osu/pull/9543. Maybe we can add an exception for tooltips.

On the topic of that area, I think the design should be improved to get all beatmaps from a playlist (like a diff range). Right now, it only gets the first beatmap's difficulty.

**Comment by bdach** [comment]
Yeah, I'd say only showing tooltips if the room is focused/selected sounds wrong.
